image-customize

```diff
@@ -77,6 +77,14 @@ class InstallAction(ActionBase):
     '''Install local rpm or distro package'''
     @staticmethod
     def execute(machine_instance, package):
+        if os.path.isfile(package):
+            pname = os.path.basename(package)
+            dest = "/var/tmp/" + pname
+            machine_instance.upload([os.path.abspath(package)], dest)
+            package = dest
```

Comment:
This should `return`, otherwise you still do the package manager detection in vain.

Reply:
True enough.  But why isn't that warning an error?

Reply:
It should be IMHO -- better than silently ignoring a broken argument.